test/fixtures/v2/documentGoerliValidWithDocumentStore.ts

```diff
@@ -63,3 +63,5 @@ export const documentGoerliValidWithDocumentStore: WrappedDocument<CustomDocumen
     merkleRoot: "9ece474bfd087cf3fc2e7fcf62f82af22cd8ae6a68ff40d7992b6335399309b4",
   },
 };
+
+
```

Comment:
Thanks for removing it and placing it in another component but can please set this file back to how it was? thanks